views/partials/footer.ejs

```diff
@@ -10,4 +12,5 @@
     <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
 </body>
+</footer>
```

Comment:
The `footer` tag should close above the `script` tag. Only after that should the `body` since it is the parent of `footer`.

Reply:
https://github.com/NJACKWinterOfCode/Instances/pull/45/commits/82955195b51b1d28f002464edf7bcc06491953f5 Done @thebishaldeb